blockchain/v2/processor_test.go

```diff
@@ -0,0 +1,236 @@
+package v2
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/tendermint/tendermint/p2p"
+	tdState "github.com/tendermint/tendermint/state"
+	"github.com/tendermint/tendermint/types"
+)
+
+// Block Response
+func TestBlockAddition(t *testing.T) {
+	var (
+		peerID         p2p.ID = "peer"
+		initHeight     int64  = 0
+		initBlock             = &types.Block{Header: types.Header{Height: initHeight}}
+		nextHeight            = initHeight + 1
+		nextBlock             = &types.Block{Header: types.Header{Height: nextHeight}}
+		tdState               = tdState.State{}
+		applicationBL         = []types.BlockID{}
+		verificationBL        = []types.BlockID{}
+		context               = newMockProcessorContext(verificationBL, applicationBL)
+		state                 = &pcState{
+			height: initHeight,
+			queue: blockQueue{
+				initHeight: queueItem{block: initBlock, peerID: peerID},
+			},
+			draining:     false,
+			blocksSynced: 0,
+			context:      context,
+			tdState:      tdState,
+		}
+		event = &bcBlockResponse{
+			peerID: peerID,
+			block:  nextBlock,
+			height: nextHeight,
+		}
+	)
+
+	nextEvent, err := state.handle(event)
+	assert.NoError(t, err, "expected no error")
+
+	assert.Equal(t, state, &pcState{
```

Comment:
```suggestion
	assert.Equal(t,  &pcState{..}, state,...
```
i.e. `assert.Equal(t, <expected>, <actual>,..)`
Occurs in other places.